qutebrowser/components/caretcommands.py

```diff
@@ -185,9 +185,14 @@ def move_to_end_of_document(tab: apitypes.Tab) -> None:
 
 @cmdutils.register(modes=[cmdutils.KeyMode.caret])
 @cmdutils.argument('tab', value=cmdutils.Value.cur_tab)
-def toggle_selection(tab: apitypes.Tab) -> None:
-    """Toggle caret selection mode."""
-    tab.caret.toggle_selection()
+@cmdutils.argument('line')
```

Comment:
No need for a `@cmdutils.argument` decorator if you don't need to specify any additional information about the argument.